src/services/TaskService.js

```diff
@@ -54,4 +54,25 @@ const postTask = async (name, description, duration, postTaskHandler) => {
   AuthService.runWithAuthToken(postTaskCallback)
 }
 
-export default { getTask, postTask };
+const taskComplete = async (taskIds, taskCompleteHandler) => {
+  let taskCompleteCallback = async function(token){
+    let body = {
+      ids: taskIds
+    }
+    let header = {
+      "Authorization": "Bearer " + token
+    }
+    let res = await axios.post('http://localhost:31337/api/tasks/complete_task', body,
+      {
+        headers:header,
+      }).then(res => console.log(res))
```

Comment:
Right here, we don't want to print the result but rather return res.data.data. You can follow the examples above